clmm/support/sampler.py

```diff
@@ -17,19 +17,15 @@ def basinhopping(model_to_shear_profile, logm_0, args) :
 
     return spo.basinhopping(model_to_shear_profile, logm_0, minimizer_kwargs={'args':args}).x[0]
 
-def scicurve_fit(profile_model,radius,profile,err_profile,bounds=None,p0=None):
+
+def scicurve_fit(profile_model,radius,profile,err_profile,**kwargs):
```

Comment:
Since the function accepts any generic keyword argument, it might be helpful to write them in the documentation. You could just copy it from `curve_fit` docs.

Reply:
Yes, but I'm also wondering about the utility of just completely wrapping `curve_fit` in `CLMM`...

Reply:
Done